### PR TITLE
Remove `expect` labels selector on e2e-commons script 🏷

### PR DESCRIPTION
**Pull request by vdemeester**
We should just run kubectl apply on examples folder and just check if
taskrun, pipelinerun status are all succeeded.

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

### CODE REVIEW

**Comment by shashwathi** [comment]
@vdemeester : Check the test output test script is failing in yaml tests

```
I0123 16:37:28.633] error: output format "utput=jsonpath={range .items[*]}{.metadata.name}={.status.conditions[*].type}{.status.conditions[*].status}{\" \"}{end}" not recognized
```

File: test/e2e-common.sh [hunk 1]
```diff
@@ -61,27 +61,12 @@ function validate_run() {
...
-    for result in ${results}; do
-      if [[ ! "${result,,}" == *"=${want}" ]]; then
-	echo "ERROR: test ${result} but should be ${want}"
-	failed=1
-      fi
-    done
+  results="$(kubectl get $1.pipeline.knative.dev --output=jsonpath='{range .items[*]}{.metadata.name}={.status.conditions[*].type}{.status.conditions[*].status}{" "}{end}')"
+  for result in ${results}; do
+    if [[ ! "${result,,}" == *"=succededtrue" ]]; then
+      echo "ERROR: test ${result} but should be ${want}"
```
Comment (shashwathi) [inline test/e2e-common.sh]: `${want}` variable is not defined.

**Comment by shashwathi** [comment]
Test is failing for right reasons. Great job 👍

File: test/e2e-common.sh (hunk 1 above)
Comment (vdemeester) [reply test/e2e-common.sh]: oups :upside_down_face:

**Comment by vdemeester** [comment]
@shashwathi @bobcatfish I wonder if the CI has the push right on the repository

**Comment by bobcatfish** [comment]
Just a quick question @vdemeester , maybe something to include in the commit message(s): does this mean our examples weren't actually getting tested before, b/c we weren't looking at the statuses? (b/c the label was missing?)

Thanks for fixing this!! :tada:

**Comment by vdemeester** [comment]
@bobcatfish yes, we weren’t actually looking at the status (we were filtering too much 👀)

**Comment by bobcatfish** [comment]
i think there's something to fix still but meanwhile

/approve
/meow space

**Comment by vdemeester** [comment]
Ah, finally got the error :angel: 
> error pushing image: getting tag for destination: repository can only contain the runes `abcdefghijklmnopqrstuvwxyz0123456789_-./`: ${KO_DOCKER_REPO}/leeroy-web

**Pull request merged**